### PR TITLE
Harleyg321 feature/temp control

### PR DESCRIPTION
**Pull request by UnchartedBull**

### CODE REVIEW

File: src/app/printer-status/printer-status.component.html [hunk 1]
```diff
@@ -1,25 +1,74 @@
...
     </tr>
-</table>
\ No newline at end of file
+</table>
+<div class="quick-control" *ngIf="view !== QuickControlView.NONE" id="quickControl">
+    <table class="top-bar" (click)="hideQuickControl()">
+        <tr>
+            <td class="top-bar__back"><img src="assets/back.svg" class="top-bar__back-icon" matRipple
+                    [matRippleUnbounded]="false" />back</td>
+            <img [src]="'assets/' + (view === QuickControlView.HOTEND ? 'nozzle.svg' : view === QuickControlView.HEATBED ? 'heat-bed.svg' : 'fan.svg')"
```
Comment (harleyg321) [inline src/app/printer-status/printer-status.component.html]: Currently the icon doesn't display. Moving this down 2 rows so the image is outside the table fixes the issue.

Alternatively, changing the class to top-bar__center-icon and keeping it where it is inside the table row gets it to display although it then isn't centered.

File: src/app/printer-status/printer-status.component.scss [hunk 2]
```diff
@@ -67,11 +67,98 @@
...
+        right: 2vw;
+        top: 3vh;
+    }
+
+    &__controller {
+        width: 35vw;
+        margin: 0 auto;
+        border: solid 0.6vw;
+        border-radius: 3vw;
+        margin-top: 5vh;
```
Comment (harleyg321) [inline src/app/printer-status/printer-status.component.scss]:
```suggestion
```
Part of the changes you've made seem to have moved the controller box down the screen, removing the top margin gets it back to the center again.

File: src/app/printer-status/printer-status.component.html [hunk 3]
```diff
@@ -1,25 +1,74 @@
...
+                class="quick-control__center-icon-small">
+        </tr>
+    </table>
+    <div class="quick-control__controller">
```
Comment (harleyg321) [inline src/app/printer-status/printer-status.component.html]:
```suggestion
            <td class="top-bar__back"><img src="assets/back.svg" class="top-bar__back-icon" matRipple
                    [matRippleUnbounded]="false" />back</td>
        </tr>
    </table>
    <img [src]="'assets/' + (view === QuickControlView.HOTEND ? 'nozzle.svg' : view === QuickControlView.HEATBED ? 'heat-bed.svg' : 'fan.svg')"
        class="quick-control__center-icon-small">
    <div class="quick-control__controller">
```

Currently the icon doesn't display. Moving rows 34/35 down 2 rows so the image is outside the table fixes the issue.

Alternatively, changing the class to top-bar__center-icon and keeping it where it is inside the table row gets it to display although it then isn't centered.

File: src/app/printer-status/printer-status.component.scss (hunk 2 above)
Comment (harleyg321) [reply src/app/printer-status/printer-status.component.scss]: Ah, it was moving the image inside the table that did it.

If you keep the image inside the table, I suggest making this change. If you move the image outside the table as in my other review comment, then this change should not be made.

**Comment by UnchartedBull** [comment]
Somehow messed it up with the icon. I moved it into the table and everything is working for me now :) Will be released in 1.4.1 (should be in the next couple of days). And once again: Closes #259

**Pull request merged**